CHANGELOG.md

```diff
@@ -15,6 +15,9 @@
   Extensions are an exception; more than one is allowed.  
   [Muhammad Zeeshan](https://github.com/mzeeshanid)
   [#2802](https://github.com/realm/SwiftLint/issues/2802)
+* Rewrite `redundant_type_annotation` rule with SwiftSyntax and add a new `ignored_attributes` option that allows disabling the rule for properties that are marked with at least one of the configured attributes
```

Comment:
You may the rule's ID and your name to the list in line 19/22 and only mention the new attribute as an enhancement.